### PR TITLE
fix(components/forms)!: remove radio button extra space on top

### PR DESCRIPTION
**Pull request by Blackbaud-SandhyaRajasabeson**
[AB#2995252](https://dev.azure.com/blackbaud/Products/_boards/board/t/SKY%20UX%20Program/Stories/?workitem=2995252)

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/2613/storybook/)

Component Storybooks:

- [action-bars-storybook](https://blackbaud.github.io/skyux-pr-preview/2613/storybooks/action-bars-storybook/)
- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/2613/storybooks/ag-grid-storybook/)
- [angular-tree-component-storybook](https://blackbaud.github.io/skyux-pr-preview/2613/storybooks/angular-tree-component-storybook/)
- [avatar-storybook](https://blackbaud.github.io/skyux-pr-preview/2613/storybooks/avatar-storybook/)
- [colorpicker-storybook](https://blackbaud.github.io/skyux-pr-preview/2613/storybooks/colorpicker-storybook/)
- [data-manager-storybook](https://blackbaud.github.io/skyux-pr-preview/2613/storybooks/data-manager-storybook/)
- [datetime-storybook](https://blackbaud.github.io/skyux-pr-preview/2613/storybooks/datetime-storybook/)
- [errors-storybook](https://blackbaud.github.io/skyux-pr-preview/2613/storybooks/errors-storybook/)
- [forms-storybook](https://blackbaud.github.io/skyux-pr-preview/2613/storybooks/forms-storybook/)
- [indicators-storybook](https://blackbaud.github.io/skyux-pr-preview/2613/storybooks/indicators-storybook/)
- [inline-form-storybook](https://blackbaud.github.io/skyux-pr-preview/2613/storybooks/inline-form-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/2613/storybooks/layout-storybook/)
- [lists-storybook](https://blackbaud.github.io/skyux-pr-preview/2613/storybooks/lists-storybook/)
- [lookup-storybook](https://blackbaud.github.io/skyux-pr-preview/2613/storybooks/lookup-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/2613/storybooks/pages-storybook/)
- [phone-field-storybook](https://blackbaud.github.io/skyux-pr-preview/2613/storybooks/phone-field-storybook/)
- [split-view-storybook](https://blackbaud.github.io/skyux-pr-preview/2613/storybooks/split-view-storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/2613/storybooks/tabs-storybook/)
- [text-editor-storybook](https://blackbaud.github.io/skyux-pr-preview/2613/storybooks/text-editor-storybook/)
- [theme-storybook](https://blackbaud.github.io/skyux-pr-preview/2613/storybooks/theme-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/2613/code-examples/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/2613/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/2613/playground/)

File: apps/playground/src/app/components/forms/radio/radio.component.html [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<sky-radio />ß
```
Comment (Blackbaud-ErikaMcVey) [inline apps/playground/src/app/components/forms/radio/radio.component.html]: Did you mean to leave this random character there?

**Review by Blackbaud-ErikaMcVey — CHANGES_REQUESTED** [review]
1. The story references radio and checkbox, these changes only apply to radio
2. I don't think they remove the space the story was talking about. When I look at the code examples build I still see extra space above the radio button like in the story screenshot 
image

File: apps/playground/src/app/components/forms/radio/radio.component.html (hunk 1 above)
Comment (Blackbaud-SandhyaRajasabeson) [reply apps/playground/src/app/components/forms/radio/radio.component.html]: nope!

**Review by Blackbaud-ErikaMcVey — APPROVED** [review]

**Pull request merged**